contracts/contracts/PreConfCommitmentStore.sol

```diff
@@ -94,13 +95,28 @@ contract PreConfCommitmentStore is OwnableUpgradeable, UUPSUpgradeable {
         uint64 decayStartTimeStamp,
         uint64 decayEndTimeStamp,
         string txnHash,
+        string revertingTxHashes,
         bytes32 commitmentHash,
         bytes bidSignature,
         bytes commitmentSignature,
         uint64 dispatchTimestamp,
         bytes sharedSecretKey
     );
 
+    /// @dev Struct for all the commitment params to avoid too deep in the stack error
+    struct CommitmentParams {
```

Comment:
The fields in `PreConfCommitment` that are duplicated in `CommitmentParams` should be removed from `PreConfCommitment`, or we can add docs on why the repeated fields need to separately exist in both structs

Reply:
Not sure I gather what you're referring to here.

Reply:
All of these fields are already a part of `PreConfCommitment` and repeated code is not ideal